MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+        {
+            timeItem.Format = "HH:mm";
+        }
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## CheckBox editor
+
+In [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) control is loaded. By default, for `bool` data type property, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor will be loaded in data form.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+**Visual states of CheckBox**
+
+CheckBox support two visual states.
+  - Checked
+  - UnChecked
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; } = false;
+
+{% endhighlight %}
+{% endtabs %}
+		
+## Switch Editor
+
+In switch editor, [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) is loaded, and DataForm [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor supports bool data type property.
+
+To add [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor in DataForm, register the editor as [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) for the required property using the [RegisterEditor]`Link` method.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+### Customizing ItemsSource of SfComboBox
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the [IDataFormSourceProvider]`Link`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string Country { get; set; }
+    
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+        public override object GetSource(string sourceName)
```

Comment:
public override object GetSource(string sourceName) => It's not override method

Reply:
override keyword removed